### ISSUE TITLE
Channel 4 xref

### ISSUE THREAD

**Issue by dXtrmn8r** [reek/anti-adblock-killer#3596]
Cross referencing last comment on #2650 in case devs can't see

![image](https://user-images.githubusercontent.com/1355374/30591369-73cae680-9d10-11e7-9841-3d900a22b750.png)

http://www.channel4.com/programmes/jamie-and-jimmys-friday-night-feast/on-demand/57010-001

Chrome + uBO + uBPX + uBPL

**Comment by uBlock-user** [comment]
Requires Flash! 

@jspenguin2017

![](https://i.gyazo.com/e8f734fb96fe663e668afbb1f5ad8334.png)

**Comment by jspenguin2017** [comment]
It's Flash so have to white list some stuff. No ads shown though. (It is in List)
```
@@/crossdomain.xml$domain=channel4.com
@@||ak.http.anno.channel4.com^$domain=channel4.com
@@||cf.http.anno.channel4.com^$domain=channel4.com
```

**Comment by dXtrmn8r** [comment]
Yeah. I already had those filters in place when I tested it and it still shows anti-adblock prompt.

**Comment by jspenguin2017** [comment]
Huh, it used to work... Well, since it is enforced in Flash, there isn't much you can do.

**Comment by uBlock-user** [comment]
Is it engraved inside the Flash player ?

**Comment by jspenguin2017** [comment]
I think so...
